src/token_kinds.jl

```diff
@@ -59,6 +59,9 @@
     begin_literal,
         LITERAL, # general
         INTEGER, # 4
```

Comment:
Should we rename this one for consistency then? `DEC_INT`?

Reply:
I think it may be worth leaving it as INTEGER for transparency purposes